src/PostgREST/ApiRequest.hs

```diff
@@ -104,7 +120,9 @@ userApiRequest schema req reqBody =
                else ActionInappropriate
           else
             case method of
-               "GET"     -> ActionRead
+               "GET"     -> case target of
```

Comment:
Maybe an if statement would be more compact rather than the sub case.

```haskell
"GET" ->
  if target == TargetRoot
    then ActionInspect
    else ActionRead
```